src/Umbraco.Core/Configuration/Models/PackageManifestSettings.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Umbraco.Cms.Core.Configuration.Models;
+
+/// <summary>
+///     Typed configuration options for package manifest settings.
+/// </summary>
+[UmbracoOptions(Constants.Configuration.ConfigGlobal)]
```

Comment:
Should this not be
```suggestion
[UmbracoOptions(Constants.Configuration.ConfigPackageManifests)]
```

Reply:
For the love of.... good thing you're awake. Copy/paste FTW. 🙈